Code/Framework/AzFramework/AzFramework/DocumentPropertyEditor/AdapterBuilder.cpp

```diff
@@ -165,6 +165,8 @@ namespace AZ::DocumentPropertyEditor
     {
         BeginLabel();
         Attribute(Nodes::Label::Value, text);
+        Attribute(Nodes::PropertyEditor::SharePriorColumn, true);
+        Attribute(Nodes::PropertyEditor::Alignment, Nodes::PropertyEditor::Align::AlignLeft);
```

Comment:
Should thees attributes be hardcoded?

Reply:
labels should definitely not share prior column by default -- that's a specialty flag that combines columns. Please contact me offline so that we can look into what issue you're trying to solve, and hopefully we can fix it on the DPE side. Ditto that alignLeft shouldn't be necessary to everything. If you can show me what it looks like with and without that, I'm sure we can make the desired effect the default in the DPE, and not something everything specifies in the DOM. Keep in mind that any attribute the builder is setting will have to be written, stored, and interpreted for every one of those nodes. Ideally, the default behavior would be the most common behavior, and specifying a particular attribute will be relatively rare.

Reply:
Honestly I'm wondering if there shouldn't just be a Prefab-specific `OverridablePropertyLabel` that handles the icon, text styling, etc rather than trying to do it with multiple handlers and dealing with layout shenanigans - should be straightforward to wrap where the ReflectionAdapter is making property labels with a virtual method you can override.

Reply:
Okay. Removed it now. We have follow up tasks to fix up ux issues with the override icon. This can be done then. I like the idea of making a virtual method and letting the DPEComponentAdapter & PrefabAdapter handle the aligning. We can even do the aligning only when overrides are present too if we want to be really specific.